### PR TITLE
More checks (`changelogCheck`, `systemlogCheck`, `thumbnailConfigCheck`), and a bit of cleanup.

### PR DESCRIPTION
**Pull request by bobdenotter**

### CODE REVIEW

File: src/EventListener/ConfigurationNoticesListener.php [hunk 1]
```diff
@@ -49,6 +55,10 @@ public function onKernelRequest(GetResponseEvent $event)
             return;
         }
 
+        $this->lotsOfRecords = $this->app['config']->get('general/configuration_notices/lotsofrecords', 10000);
+
+        $this->app['stopwatch']->start('bolt.configuration_notices');
```
Comment (GawainLynch) [inline src/EventListener/ConfigurationNoticesListener.php]: :heart: … Serious amounts of "thank you" for remembering this one! :+1:

File: README.md [hunk 2]
```diff
@@ -14,3 +14,18 @@ composer require bolt/configuration-notices
 ```
 
 ![7e22d8c8-bbdd-11e6-9668-48a1ec86ba04](https://cloud.githubusercontent.com/assets/1833361/21287029/3f4d8cf4-c463-11e6-8cd1-69583e7fa2ba.png)
+
+You can influence a few of the checks through the configuration. How meta! 
+
+```yaml
+configuration_notices:
+    local_domains: [ '.localhost' ]
+    lotsofrecords: 100
```
Comment (GawainLynch) [inline README.md]: `lotsofrecords` and the use of the phrase in function names is confusing, and semantically wrong.

What about `log_threshold: 100` … I immediately thought "to use "limit", but it is not a limit, it is just a threshold at which point you will warn the log is getting too big

Comment (bobdenotter) [inline README.md]: I had the same thoughts.. "Limit" didn't sound right. I'll replace with `log_threshold`. 👍

**Comment by bobdenotter** [comment]
Requested changes done.

**Pull request merged**